lib/ansible/modules/database/mysql/mysql_user.py

```diff
@@ -115,87 +115,74 @@
     name: ''
     host: localhost
     state: absent
-
 - name: Removes all anonymous user accounts
   mysql_user:
     name: ''
     host_all: yes
     state: absent
-
 - name: Create database user with name 'bob' and password '12345' with all database privileges
```

Comment:
@azeliashchonak 
Many whitespace changes there, probably introduced by automatic formatting.
Could you revert those changes, unless they are necessary / related to the object of the current PR ?

Reply:
Why should we leave empty rows in comments?

Reply:
Oh ! We both misread / misunderstood theses changes: they are not whitespace changes, neither empty rows.
Removing these lines completely changes the structure, removing one level of list (list of lists becomes simple list).
I don't know if this extra level is necessary or not, we need to check how documentation should be provided.

@felixfontein @gundalow Do you have any input about this ?

Reply:
@azeliashchonak this is not a comment, but extracted and put into the documentation of the module. See here: https://docs.ansible.com/ansible/devel/modules/mysql_user_module.html#examples

In general, examples are better readable if different examples are separated with empty lines. So I would definitely keep them.

Reply:
I've got it, Thanks. Fixed.